etc/skydive.yml.default

```diff
@@ -104,6 +104,11 @@ agent:
   # Not required, but can be used to allow virtual hosting
   # X509_servername: domain.com
   #
+  http:
+    # log the HTTP client request and response (to log level DEBUG)
+    # debug: false
+    # define the Cookie HTTP Request Header
+    # cookie: { "<name1>": "<value1>", "<name2>": "<value2>" }
```

Comment:
I would prefer to keep the file using yaml syntax only so

```
cookie:
  name1: value1
  name2: value2
```

Reply:
ok, have changed the example in the skydive.yml.default (no code change needed)